src/lib/serviceRoutinesV2/getEntityAttributeValue.cpp

```diff
@@ -103,13 +104,21 @@ std::string getEntityAttributeValue
       // Do not use attribute name, change to 'value'
       attribute.pcontextAttribute->name = "value";
 
-      TIMED_RENDER(answer = attribute.render(ciP, EntityAttributeValueRequest, false));
+      TIMED_RENDER(answer = attribute.render(ciP->apiVersion,
+                                             ciP->httpHeaders.accepted("text/plain"),
+                                             ciP->httpHeaders.accepted("application/json"),
+                                             ciP->httpHeaders.outformatSelect(),
+                                             &(ciP->outMimeType),
+                                             &(ciP->httpStatusCode),
+                                             ciP->uriParamOptions[OPT_KEY_VALUES],
+                                             ciP->uriParam[URI_PARAM_METADATA],
+                                             EntityAttributeValueRequest, false));
     }
```

Comment:
Look at this, a very very long TIMED_RENDER line, and yet, **so** easy to read.
Each param on a separate line and no param excessively long.
Just the way it should be!

Ah, except the last `false` ... :-(